src/main/java/woowa/bossdog/subway/domain/BaseEntity.java

```diff
@@ -0,0 +1,22 @@
+package woowa.bossdog.subway.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(value = {AuditingEntityListener.class})
+public class BaseEntity {
```

Comment:
`BaseEntity`는 인스턴스화가 될 수 없도록 추상 클래스로 만들면 어떨까요?

```suggestion
public abstract class BaseEntity {
```